ShenkinStore/Controllers/ShoppingCartController.cs

```diff
@@ -0,0 +1,106 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using ShenkinStore.Models;
+using ShenkinStore.ViewModels;
+
+namespace ShenkinStore.Controllers
+{
+    public class ShoppingCartController : Controller
+    {
+        ShenkinContext db = new ShenkinContext();
+
+        public string shoppingCartId { get; set; }
+
+
+        // GET: ShoppingCart
+
+        public ActionResult Index()
+        {
+         
+            var userID = HttpContext.Session.GetString("UserID");
+            
+            
+            
+            
+            if (userID != null)
+            {
+                var cart = ShoppingCart.GetCart(userID.ToString());
+
+                var viewModel = new ShoppingCartViewModel
+                {
+                    CartItems = cart.GetCartItems(),
+                    CartTotal = cart.GetTotal()
+                };
+                return View(viewModel);
+            }
+            else
```

Comment:
Please add {} on all the else.
In case someone will add code there it won't work (because without {} it only activate the first row).

Reply:
Ok